### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AryyNewbie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ See `dontbug record --help` for more information on the various flags available
 ## Command `dontbug replay`
 The `dontbug replay` command replays a previously saved execution trace to a PHP IDE debugger. You may set breakpoints, step through code, inspect variable values etc. as you are used to. But more interestingly, dontbug allows you to _reverse_ debug i.e. step over backwards, run backwards, hit breakpoints when running in reverse and so forth.
 
-dontbug communicates with PHP IDEs by using the [dbgp](https://xdebug.org/docs-dbgp.php) protocol which is the defacto standard for PHP IDEs so _no special support_ is required for dontbug to work with them. As far as the IDEs are concerned they are talking with a normal PHP debug engine.
+dontbug communicates with PHP IDEs by using the [dbgp](https://xdebug.org/docs-dbgp.php) protocol which is the defactor standard for PHP IDEs so _no special support_ is required for dontbug to work with them. As far as the IDEs are concerned they are talking with a normal PHP debug engine.
```
Comment (sidkshatriya) [inline README.md:129]: defacto is actually correct here :-)

File: README.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ The `<php-source-root-dir>` means the outermost directory of all possible PHP sc
 - If you have sources symlinked from inside the `<php-source-root-dir>` to outside that dir, dontbug should be able to handle that (without you having to increase the scope of the `<php-source-root-dir>`)
 
 ### PHP built-in webserver tips
-You may record as many http page loads for later debugging when running the PHP built in webserver (unlike traditional PHP debugging which is usually one page load at a time). **However be aware that recording too many page loads may degrade performance during debugging**. Additionally, you may _not_ pass arguments to the PHP built in server i.e. the `--args` flag is ignored if not used in conjunction with `--php-cli-script`.
+You may record as many http page loads for later debugging when running the PHP built in webserver (unlike traditional PHP debugging which is usually one page load at a time). **However be aware that recording too many pages loads may degrade performance during debugging**. Additionally, you may _not_ pass arguments to the PHP built in server i.e. the `--args` flag is ignored if not used in conjunction with `--php-cli-script`.
```
Comment (sidkshatriya) [inline README.md:109]: "page loads" is actually correct here... :-)

**Comment by AryyNewbie** [comment]
i am sorry about that, i trying the languagetool.org

2016-10-20 14:16 GMT+08:00 Sidharth Kshatriya notifications@github.com:

**Comment by AryyNewbie** [comment]
i corect many page to many pages

2016-10-20 14:20 GMT+08:00 Aryy Newbie aryy.newbie@gmail.com:

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Dontbug Debugger
 
-Dontbug is a reverse debugger (aka _time travel debugger_) for PHP. It allows you to record the execution of PHP scripts (in command line mode or in the browser) and replay the same execution back in a PHP IDE debugger. During replay you may debug normally (forward mode debugging) or in reverse, which allows you to step over/out backwards, step backwards, run backwards, run to cursor backwards, set breakpoints in the past and so forth.
+Dontbug is a reverse debugger (aka _time travel debugger_) for PHP. It allows you to record the execution of PHP scripts (in command line mode or in the browser) and replay the same execution back in a PHP IDE debugger. During replay, you may debug normally (forward mode debugging) or in reverse, which allows you to step over/out backwards, step backwards, run backwards, run to cursor backwards, set breakpoints in the past and so forth.
```
Comment (sidkshatriya) [inline README.md:3]: This is stylistic choice. Sometimes you want to have a comma to allow the readers to pause but sometimes you don't need to. There is a comma after "reverse" which pauses the reader. Sometimes having too many commas breaks the flow of the prose.

Thanks for the suggestion though. I may add a comma here as you've suggested but I think its OK for now...

**Comment by AryyNewbie** [comment]
ok..

2016-10-20 14:24 GMT+08:00 Sidharth Kshatriya notifications@github.com:

**Pull request closed without merging**